### PR TITLE
Adds powercreep to Supply Borgs

### PR DESCRIPTION
**Pull request by jknpj**
:cl:
 * rscadd: Ports the Cyborg GPS from TEEGEE and adds it to Supply Cyborgs
 * tweak: Supply Cyborgs mining satchel is now called 'automatic ore loader'. It can be toggled on to automatically pick up AND insert ore inside a nearby ore box(Sprite by Blithering).

### CODE REVIEW

**Comment by Kurfursten** [comment]
The GPS, sure, but what's the reasoning behind just giving cyborgs a better bag and not making it available to humans?

**Comment by jknpj** [comment]
Supply borgs don't get a drill upgrade so this is to make them have their own niche between the fast ddrill miner and the multi-tile instant loader ripley.

**Comment by Kurfursten** [comment]
👌  Makes sense enough then

**Comment by FudgePucker** [comment]
I like playing as a mining borg. You can get a load of silver gold uranium and diamonds, six Goliath corpses and the polar star/spur parts, all before the miners can put on their hardsuits and get out there. Not an exaggeration, that's what happened last time. Too bad you don't really have a use for it as a cyborg, you just do it for the thrill of it.

File: code/game/objects/items/weapons/storage/bags.dm [hunk 1]
```diff
@@ -110,6 +110,72 @@ obj/item/weapon/storage/bag/plasticbag/quick_store(var/obj/item/I)
...
+			if(ore.material)
+				remove_from_storage(ore)
+				box.materials.addAmount(ore.material, 1)
+				qdel(ore)
+
+/obj/item/weapon/storage/bag/ore/auto/proc/mob_moved(var/list/event_args, var/mob/holder)
+	if(isrobot(holder))
+		var/mob/living/silicon/robot/S = holder
+		if(locate(src) in S.get_all_slots())
+			if(handling)
```
Comment (DamianX) [inline code/game/objects/items/weapons/storage/bags.dm]: This proc should never get called if handling is false, right?

**Pull request merged**